src/3d/framegraph/qgsdepthrenderview.h

```diff
@@ -0,0 +1,78 @@
+/***************************************************************************
+  qgsdepthrenderview.h
+  --------------------------------------
+  Date                 : august 2022
```

Comment:
is this correct?

Reply:
sadly! It was the first test we made a long time ago